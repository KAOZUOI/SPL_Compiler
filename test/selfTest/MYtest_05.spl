struct Employee {
  int id;
  float salary;
};
int main() {
  struct Employee emp1;
  emp1.id = 101;
  emp1.salary = 50000.0;
  emp1.department = 'S';
  return 0;
}
