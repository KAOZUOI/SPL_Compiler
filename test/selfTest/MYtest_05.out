Error type 14 at Line 9: the structure does not have the field "department".
Error type 5 at Line 9: unmatching types appear at both sides of the assignment operator (=) 